src/App/helpers/encryption.js

```diff
@@ -0,0 +1,8 @@
+import bcrypt from 'bcryptjs';
```

Comment:
This file has not been used! Please make sure to use it instead of what you did in your controllers!

Reply:
@Procech20 why did you resolve this comment without fixing the issue?